src/test/java/com/socrata/ApiTest.java

```diff
@@ -36,32 +37,23 @@
  */
 public class ApiTest extends TestBase
 {
-
-
     /**
-     * Tests a number of ways to connect using 4x4s
-     *
-     * Will issue a simple query, and do spot validation.
-     */
+     * Tests a number of ways to connect using 4x4. Will issue a simple query, and do spot validation. */
     @Test
-    @Ignore // TODO Dataset isn't present, change tests to create dataset
     public void testSimpleQuery4x4() throws LongRunningQueryException, SodaError, InterruptedException, IOException
     {
         final HttpLowLevel  connection = connect();
         final SodaDdl       sodaDdl= new SodaDdl(connection);
 
-        final SearchResults results =  sodaDdl.searchViews(new SearchClause.NameSearch("Test Data"));
+        final SearchResults results =  sodaDdl.searchViews(new SearchClause.NameSearch("SodaTestData"));
```

Comment:
Did we lose the original "Test Data" dataset?

Reply:
We lost all of them--I couldn't find any static datasets from any of the tests on https://sandbox.demo.socrata.com. 😞

Reply:
Is it simple to recreate the same dataset?

Reply:
Recreating the same dataset takes research, time, and refactoring. I recreated the dataset for this and another test `testSearch` in SodaDdlTest, but I did not recreate datasets for other tests that require static assets. That work will be captured [here](https://socrata.atlassian.net/browse/EN-45878).